scripts/patterns-reorg/main.py

```diff
@@ -37,6 +37,7 @@ def write_guides_dir() -> None:
         "children": [entry.to_json("guides") for entry in TOP_LEVEL_ENTRIES],
     }
     text = json.dumps(result, indent=2)
+    text = text.replace('"url": "/guides/index"', '"url": "/guides"')
```

Comment:
I think a better fix would be in `models.py`, probably with how we store the slug in the first place. For example, allow `slug=""` for the index. (That would mean you change other slugs to be `slug="/my-page"`, rather than `slug="my-page"`

We want both this Toc.json thing _and_ redirects to be using `/guides` rather than `/guides/index`. So it's better to fix this at the source.